openthread/lib/cortex-m33/hard-float/v1.3/ftd/oberon/openthread_lib_configuration.txt

```diff
@@ -1,17 +1,16 @@
-GCC_version: arm-zephyr-eabi-gcc (Zephyr SDK 0.15.2) 12.1.0
+GCC_version: arm-zephyr-eabi-gcc (Zephyr SDK 0.16.0) 12.2.0
```

Comment:
Now I realize, the recommended Zephyr SDK version is 0.16.1 here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation/installing.html#install-the-zephyr-sdk

Shouldn't the libraries be built with the same version?

Reply:
Incorrect docker was used, libs will be rebuilt.